runtime/src/bank.rs

```diff
@@ -556,6 +557,17 @@ impl Bank {
         self.store_account(pubkey, &new_account);
     }
 
+    fn update_incinerator(&self) {
+        let account = self.get_sysvar_account(&sysvar::incinerator::id());
```

Comment:
how about using `update_sysvar_accont` by making it expect `Option<Account>` from the closure?

Reply:
I considered that but It would cause a large amount of code churn for only this one use case and didn't seem to be worth it.